modules/json/pom.xml

```diff
@@ -36,6 +47,12 @@
     <dependency>
       <groupId>com.fasterxml.jackson.datatype</groupId>
       <artifactId>jackson-datatype-jsr310</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>com.fasterxml.jackson.core</groupId>
+          <artifactId>jackson-databind</artifactId>
```

Comment:
if we succeed with my above suggestions, this can also be stripped and we get rid of such bells and whistles that are bad for maintenance.